parser_edu.go

```diff
@@ -0,0 +1,47 @@
+package whoisparser
+
+import (
+	"regexp"
+)
+
+var eduParser = &Parser{
+
+	errorRegex: &ParseErrorRegex{
+		NoSuchDomain:     regexp.MustCompile(`NO MATCH`),
+		RateLimit:        nil,
+		MalformedRequest: regexp.MustCompile(`Invalid domain name`),
+	},
+
+	registrarRegex: &RegistrarRegex{
+		CreatedDate:    regexp.MustCompile(`Domain record activated: *(.+)`),
+		DomainDNSSEC:   nil,
```

Comment:
If there is no special reason to explicitly set field to nil the field should be omitted the way it is done in `adminRegex`. Otherwise please leave a comment with some explanation.